main.py

```diff
@@ -148,5 +149,76 @@ async def keen(context):
     await context.send(result)
 
 
-bot.run(TOKEN)
+@bot.command(name='daily', help='Will give the daily leaderboard for specified day')
+async def daily(context, day : str = str((datetime.datetime.today() - datetime.timedelta(hours=5)).day)):
+    # The default day is whatever day's challenge has just come out. So at 4.59AM UTC will still show previous day's leaderboard
```

Comment:
Untested fix for this giving the incorrect day:
```
async def daily(context, day : str = None):
    # The default day calculation cannot be in the function default value because the default value is evaluated when the program is started, not when the function is called
    if day is None:
        # The default day is whatever day's challenge has just come out. So at 4.59AM UTC will still show previous day's leaderboard
        str((datetime.datetime.today() - datetime.timedelta(hours=5)).day)
```